cirq-core/cirq/vis/circuit_to_latex_quantikz.py

```diff
@@ -0,0 +1,710 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please add the Apache copyright header.  See any other files in cirq for an example.